test/ar_flexmaster_test.rb

```diff
@@ -221,7 +221,7 @@ def test_recovering_after_losing_connection_to_the_master
       User.create!
     end
 
-    assert_equal nil, User.connection.instance_variable_get("@connection")
+    assert_nil User.connection.instance_variable_get("@connection") if ActiveRecord::VERSION::STRING < '5.1'
```

Comment:
Should we assert something else for Rails 5.1? Can we assert that the connection is closed?